cub/cub/device/dispatch/tuning/tuning_transform.cuh

```diff
@@ -115,11 +117,10 @@ _CCCL_HOST_DEVICE constexpr auto loaded_bytes_per_iteration() -> int
   return (int{sizeof(it_value_t<Its>)} + ... + 0);
 }
 
-constexpr int bulk_copy_alignment     = 128;
```

Comment:
question: this doesn't seem like a tuning parameter to me. Given how you specify one value or another purely on the SM architecture, it feels more like hardware property. If you don't want to tune this value, can we make this a device function with `NV_IF_TARGET` instead to simplify the code and remove dependencies on `__CUDA_ARCH_LIST__` in tuning etc.? The only place you need this at compile time is static assert, but you have equivalent check elsewhere with `_CCCL_ASSERT_HOST`.

Reply:
Agreed